### PR TITLE
Relicense under the MIT license

### PR DESCRIPTION
**Pull request by nsoranzo**
The Academic Free License is not compatible with the Debian Free Software Guidelines; relicensing under the MIT license will allow Planemo to be packaged by Debian.

See also https://www.gnu.org/licenses/license-list.html#AcademicFreeLicense

### CODE REVIEW

**Comment by nsoranzo** [comment]
Ping @nturaga @shiltemann @mr-c

**Review by bgruening — APPROVED** [review]

**Review by martenson — APPROVED** [review]

**Comment by nsoranzo** [comment]
> Ideally we would get every single contributor to agree, although if there is a minimum threshold is perhaps open to debate?

Once we get approval from the more frequent contributors, I'll ping more people from https://github.com/galaxyproject/planemo/graphs/contributors .

**Review by lorrainealisha75 — APPROVED** [review]

**Comment by nsoranzo** [comment]
@mr-c Can you formally approve please? You have contributed a couple of commits some time ago.

**Comment by nturaga** [comment]
+1 from end.

**Review by hexylena — APPROVED** [review]

**Review by bebatut — APPROVED** [review]

**Review by shiltemann — APPROVED** [review]

**Review by jmchilton — APPROVED** [review]

**Comment by nsoranzo** [comment]
Ping @abretaud @kellrott @malloryfreeberg @AndreasSko : can you please approve this pull request if you agree with the license change?

**Review by malloryfreeberg — APPROVED** [review]

**Review by kellrott — APPROVED** [review]

**Review by abretaud — APPROVED** [review]

**Review by AndreasSko — APPROVED** [review]

**Review by natefoo — APPROVED** [review]

**Review by davebx — APPROVED** [review]

**Review by mr-c — APPROVED** [review]

**Review by blankenberg — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks all for the approvals, I think we have all substantial contributors on board. :tada: 

@jmchilton Are you OK to merge?

**Comment by jmchilton** [comment]
Thanks for leading the charge @nsoranzo - I really appreciate your efforts here.

**Pull request merged**
